views/partials/navigation.pug

```diff
@@ -15,6 +15,8 @@ nav(class="navbar navbar-expand-lg navbar-light bg-light")
           a(class="nav-link", href="/training") Training
       li(class="nav-item")
         a(class="nav-link", href="/apply") Apply
+      li(class="nav-item")
+        a(class="nav-link", href="/booklet") Info Booklet
```

Comment:
Is the info booklet meant to be seen for everyone even if they are not logged in? The only reason I'm asking is because the old info book was hidden under "training"  which was only seen by logged in users.

Reply:
I thought so but I asked Ellie and she agreed with you so I made those changes!